lib/board.rb

```diff
@@ -2,32 +2,58 @@ class Board
   attr_reader :grid
 
   def initialize
-    @grid = top_row
+    @grid = numbered_row
```

Comment:
I'm a little confused to see `@grid` being set here, then the next line is `build_grid`. Could this line be a part of that method?

Reply:
I like that idea a lot. I was tied to "starting" to build the grid in initialize but I agree that it makes more sense for all the logic to live in one method.